tests/test_kdtree.cpp

```diff
@@ -209,18 +209,6 @@ TEST_CASE("All triangles are in the same plane", "[kdtree]") {
     }
 }
 
-TEST_CASE("Degenerated triangles test", "[KDTree]") {
```

Comment:
This test can't pass anymore, since our new geometry classes don't want to work in debug mode with NaN floats by design. Since the triangles are degenerated, their normals are NaN.